css-size-adjust-1/Overview.bs

```diff
@@ -97,54 +108,90 @@ The default size adjustment affects text and form controls, whether
 those form controls contain text (e.g., text inputs, selects) or do not
 (e.g., radio buttons, checkboxes).
 
-Conditions that suppress adjustment {#default-adjustment-conditions}
---------------------------------------------------------------------
+Adjustment Conditions {#default-adjustment-conditions}
+------------------------------------------------------
+
+A User Agent may consider any information when determining which elements
+get adjusted.
 
-A number of conditions suppress the default adjustment because these
-conditions are associated with layouts for which the user experience
-would be worsened by size adjustment rather than improved by it.  These
-conditions are:
+Note: In general, User Agents are trying to determine which elements are
+integral to the structure of the page, and thus shouldn't be adjusted,
+and which elements are allowed to flow freely and thus can be adjusted
+without undesirable rendering artifacts.
 
+Here are some criteria which User Agents may consider when making this
+determination:
 <ul>
-  <li>when the total amount of text in the block formatting context (see
+  <li>When the total amount of text in the block formatting context (see
```

Comment:
technically I think this should probably still be lowercase since they're not complete sentences.  (They can still end with a period, since they're *followed* by complete sentences in many cases.)